### ISSUE TITLE
Encode Licenses and Copyrights

### ISSUE THREAD

**Issue by shnizzedy** [ChildMindInstitute/mindlogger-app-backend#94]
https://mindlogger.info/faq/ says

> However, while we are working on the following, the current version of Mindlogger:
> [. . .]
> - does not handle proprietary questionnaires or other content in its library

However, nowhere in our platform do we currently encode, check for, enforce or even ask about intellectual property rights. I fear and suspect that most of the content that we're currently exposing is in violation of the various instruments' assigned licenses. We should provide links to documentation of and/or the full text of a given entity's copyright/license, and I think these should be

- required at the top (applet/activity set) level,
- optional at the lower (activity, screen) and atomic (question text, option text, stimulus image) levels, and
- implicitly inherited from an entity's parent when not included.  

I took a stab at encoding licenses in [TheColumbiaImpairmentScale-SelfReportVersion_schema](https://github.com/ReproNim/schema-standardization/blob/4ab8c05/activities/TheColumbiaImpairmentScale-SelfReportVersion/TheColumbiaImpairmentScale-SelfReportVersion_schema.jsonld) (which, as far as I could tell is copyrighted) in [ReproNim/schema-standardization](https://github.com/ReproNim/schema-standardization) format in the development database:

#### https://mindlogger-dev.vasegurt.com/api/v1/folder/5c6ae0ea9c7ea529fbe2459e
```json
{
  "_accessLevel": 2,
  "_id": "5c6ae0ea9c7ea529fbe2459e",
  "_modelType": "folder",
  "baseParentId": "5c6aec8e9c7ea5529ee58db2",
  "baseParentType": "collection",
  "created": "2019-02-18T16:44:26.713000+00:00",
  "creatorId": "5c65e1c09c7ea529fbe24594",
  "description": "",
  "meta": {
    "@context": [
      "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/contexts/generic.jsonld",
      "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/TheColumbiaImpairmentScale-SelfReportVersion/TheColumbiaImpairmentScale-SelfReportVersion_context.jsonld"
    ],
    "@id": "TheColumbiaImpairmentScale-SelfReportVersion_schema",
    "@type": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/schemas/Activity.jsonld",
    "preamble": "This survey asks questions about how you think you are doing overall. Please answer questions on a scale from 0 to 4 with 0 being \"No Problem\" to 4 being \"Very Bad Problem\"",
    "schema:copyrightHolder": "https://www.wiley.com",
    "schema:copyrightYear": 1993,
    "schema:description": "TheColumbiaImpairmentScale-SelfReportVersion schema",
    "schema:license": "https://eprovide.mapi-trust.org/instruments/columbia-impairment-scale",
    "schema:schemaVersion": "0.0.1",
    "schema:version": "0.0.1",
    "skos:altLabel": "TheColumbiaImpairmentScale-SelfReportVersion_schema",
    "skos:prefLabel": "TheColumbiaImpairmentScale-SelfReportVersion_schema",
    "ui": {
      "order": [
        "CIS_SR_01",
        "CIS_SR_02",
        "CIS_SR_03",
        "CIS_SR_04",
        "CIS_SR_05",
        "CIS_SR_06",
        "CIS_SR_07",
        "CIS_SR_08",
        "CIS_SR_09",
        "CIS_SR_10",
        "CIS_SR_11",
        "CIS_SR_12",
        "CIS_SR_13"
      ],
      "shuffle": false
    }
  },
  "name": "Columbia Impairment Scale (CIS)",
  "parentCollection": "folder",
  "parentId": "5c6ae0de9c7ea529fbe2459d",
  "public": false,
  "size": 0,
  "updated": "2019-02-18T17:08:48.337000+00:00"
}
```

I pulled in 3 IP-related terms from schema.org. @sanuann, what do you think about using these terms? @anirudh4792, do you have a good sense of where the licenses and copyrights of the assessments can be found, ie, how did we split those labeled "proprietary" and "non-proprietary"?

Related: #2, #3, #32, https://github.com/ChildMindInstitute/mindlogger-app/issues/42

**Comment by satra** [comment]
we should use `schema:isProprietary`, which if `True` would then require the manager to acknowledge license rights.

and use `schema:license` to point to the appropriate URL

i like the additions of copyright holder and copyright year

**Comment by shnizzedy** [comment]
@satra, I agree in principle, but I think [`schema:isProprietary`](https://health-lifesci.schema.org/isProprietary) is too specific a property:

> True if this item's name is a proprietary/brand name (vs. generic name).

This property is probably applicable to a nontrivial number of activities, but plenty of generically-named components will also require license acknowledgment.

**Comment by satra** [comment]
@shnizzedy - i think we want to isolate cases where mindlogger or schema-ui would need to acknowledge permission from the license/copyright holder. and i was thinking of `isProprietary` as the boolean for that.

if there is a better key that would be good, but wanted to stick to things in schema.org if possible.

**Comment by shnizzedy** [comment]
How about [`schema:isAccessibleForFree`](https://schema.org/isAccessibleForFree):

> A flag to signal that the item, event, or place is accessible for free.

, which if `False` would then require the manager to acknowledge license rights (though it would be good to acknowledge license rights regardless)?
